File: Dart/Directory/check_directory_exists.dart
import "dart:core";
import "dart:io";

void main(){
  // Check if the directory exists.
  print(checkDirectoryExists("assets/valid/j3U5uJY779L49q98MX86iFsxs2kY9ew3/"));
}

// Check if a directory exists
bool checkDirectoryExists(final String path) {
  return FileSystemEntity.typeSync(path) != FileSystemEntityType.notFound;
}

File: Python/Directory/check_directory_exists.py
import os


# Check if a given directory exists.
def check_directory_exists(system_path):
    return os.path.exists(system_path)


def main():
    # Check if a given directory exists.
    print(check_directory_exists("assets/valid/j3U5uJY779L49q98MX86iFsxs2kY9ew3"))


main()
